lib/read-docs.js

```diff
@@ -11,10 +11,18 @@ module.exports = function readDocs (docName) {
 
   return folders.map(function (docs) {
     var version = path.basename(path.dirname(docs)).replace('v', '')
+    var data
+
+    try {
+      data = JSON.parse(fs.readFileSync(docs, { encoding: 'utf8' }))
+    } catch (e) {
+      console.error(docs)
```

Comment:
i think we want to `process.exit(1)` here after logging. I don't think we want the program to continue running with invalid data.